ilastik.py

```diff
@@ -77,6 +77,7 @@ def main():
     # Provide your command-line args here. See examples below.
     
     ## Auto-open an existing project
+    parsed_args.project='/home/andreas/MyProject.ilp'
```

Comment:
Could we remove all of these lines? Or leave just one in as an example, but not add more?